route/people-route.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const Router = require('express').Router;
+const jsonParser = require('body-parser').json();
+const createError = require('http-errors');
+const People = require('../model/people.js');
+const peopleRouter = module.exports = new Router();
+const debug = require('debug')('people:people-route');
+
+peopleRouter.post('/api/people', jsonParser, function(req, res, next) {
+  req.body.timestamp = new Date();
+  new People(req.body).save()
+  .then(people => res.json(people))
+  .catch( err => next(err));
```

Comment:
You need some kind of validation catch here otherwise if you try and pass an invalid `req.body` mongo will throw a validation error but you don't have anything to catch it with to make sure it doesn't default to a 500. You can choose to catch that error here or in your error middleware you just want to make sure you do catch it correctly and pass the right error status and message since a 500 isn't correct.